benchmarks/torchbench_model.py

```diff
@@ -29,6 +29,27 @@
     "detectron2_fcos_r_50_fpn",
 }
 
+# torchbench models that might OOM using Adam.
+# This list was extracted from PyTorch's repository: benchmarks/dynamo/common.py
```

Comment:
So in the link it is mentioned that they use Adam optimiser in the CI. What optimiser is used in the official PyTorch HUD? I'm asking because we want to be as close as possible.

Reply:
Good catch. So, maybe we want to get the previous set: `BENCHMARK_USE_SGD`, which is used outside CI.